### DISCUSSION TITLE
Reset Stellarium by script ...

### DISCUSSION THREAD

**Discussion by StellariumSentinel** [Stellarium/stellarium#3640] [Q&A]
Hi everyone! 
Is there a script or command line script capable of resetting stellarium to its initial state (reposition the initial viewing horizon, turn off active constellations, etc.)? 

I mean what it looks like on startup...

Thank you all!

**Comment by gzotti** [comment]
No, such feature is not available.

**Comment by alex-w** [comment]
See `core.clear();` function, but this is very partial use case

**Reply by gzotti** [reply]
That sets a predefined state, not (necessarily) the user's settings from config.ini.
